web/Services/LandlordService.cs

```diff
@@ -115,7 +115,7 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
 
     public Task<LandlordDbModel?> GetLandlordIfExistsFromId(int? id)
     {
-        return _dbContext.Landlords.SingleOrDefaultAsync(l => l.Id == id);
+        return _dbContext.Landlords.Include(l=>l.Properties).SingleOrDefaultAsync(l => l.Id == id);

```

Comment:
This will load all of the landlord's properties into memory from the database, which is fine when we need them, but most of the time this method is called we don't.

So can you either make this a separate method (`GetLandlordIfExistsWithProperties`) or load the (count of the) properties separately?